router/router.nginx.conf.in

```diff
@@ -6,7 +6,7 @@ map $http_x_forwarded_proto $updated_scheme {
 }
 
 server {
-    server_name @domain@ www.@domain@;
+    server_name @domain@ www.* site.*;
```

Comment:
Do you still need domain here? If not, does it need to be taken out of the templating?

Reply:
I think I do, because we route http://hail.is/ to this server block.